build/params_2k.go

```diff
@@ -56,7 +56,7 @@ var UpgradeOhSnapHeight = abi.ChainEpoch(-18)
 
 var UpgradeSkyrHeight = abi.ChainEpoch(-19)
 
-var UpgradeV17Height = abi.ChainEpoch(99999999999999)
+var UpgradeSharkHeight = abi.ChainEpoch(100)
```

Comment:
Can we keep this consistent with other upgrades in 2k devnets and set this to `-20`?

Reply:
Eventually? I like starting on the prev version, and upgrading early as we're testing a new nv. After Shark sharks, I would support bumping devnet to starting from v17 by default.